builtin/logical/pki/cert_util_test.go

```diff
@@ -638,11 +638,10 @@ func testParseCertificateToFields(t *testing.T, issueTime time.Time, tt *parseCe
 			require.NoError(t, err)
 
 			diff := expectedTTL - actualTTL
-			if diff < 0 {
-				diff = -diff
-			}
-			require.LessOrEqual(t, diff, 1*time.Second,
-				"ttl must be at most 1s off, want: %s got: %s", tt.wantFields["ttl"], fields["ttl"])
+			require.GreaterOrEqual(t, diff, 0,
+				"ttl should be, if off, smaller than expected want: %s got: %s", tt.wantFields["ttl"], fields["ttl"])
+			require.LessOrEqual(t, diff, 30*time.Second, // Test can be slow, allow more off in the other direction
+				"ttl must be at most 30s off, want: %s got: %s", tt.wantFields["ttl"], fields["ttl"])
```

Comment:
Sine the not before durations are around 30s, maybe use a smaller tolerance? Shouldn't 10s be enough?